sigs.yaml

```diff
@@ -1503,6 +1503,7 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/custom-metrics-apiserver/master/OWNERS
   - name: heapster
+    description: "(Deprecated)"
```

Comment:
Should we also deprecate mutating-trace-admission-controller per https://github.com/kubernetes/org/issues/2527 ?

Reply:
removed